configure.ac

```diff
@@ -74,9 +74,15 @@ AC_CHECK_HEADER(thenonexistentheader.h, HAVE_THENONEXISTENTHEADER_H="true")
 
 # set some ld -R nonsense
 #
+AC_ARG_ENABLE(rpath,
+[  --disable-rpath         do not hardcode runtime library paths],
+:, enable_rpath=yes)
```

Comment:
> And therefore that action-if-not-given clause will be active iff --enable-rpath was not given.

Just to clarify: I believe you want that clause to be active iff `--diable-rpath` was not given. That is the opposite.